common/src/main/scala/org/apache/celeborn/common/rpc/netty/NettyRpcEnv.scala

```diff
@@ -390,8 +390,13 @@ private[celeborn] class NettyRpcEnvFactory extends RpcEnvFactory with Logging {
       new JavaSerializer(celebornConf).newInstance().asInstanceOf[JavaSerializerInstance]
     val nettyEnv = new NettyRpcEnv(config, javaSerializerInstance)
     val startNettyRpcEnv: Int => (NettyRpcEnv, Int) = { actualPort =>
-      logInfo(s"Starting RPC Server [${config.name}] on ${config.bindAddress}:$actualPort " +
-        s"with advisor endpoint ${config.advertiseAddress}:$actualPort")
+      if (celebornConf.bindWildcardAddress) {
+        logInfo(s"Starting RPC Server [${config.name}] on wildcard address with port" +
```

Comment:
`wildcard address with port` means - is it something like `*:8080`?

Reply:
Looks like it will be `0.0.0.0` in `IPv4` case and `::0` in `IPv6` to signify unspecified or invalid IP address.

Reply:
Just a suggestion: it would make it clearer for others if added as a `static final WILDCARD_ADDRESS = null` and use it in both `HttpService` and `RpcEnv` as well with a comment explaining what it is for.